scene/gui/code_edit.cpp

```diff
@@ -1400,15 +1400,16 @@ void CodeEdit::fold_line(int p_line) {
 	}
 
 	/* Find the last line to be hidden. */
-	int end_line = get_line_count();
+	int last_line = get_line_count() - 1;
```

Comment:
Would suggest renaming this to `line_count` or  `text_size` to be little clearer. Could also make it `const`.